src/Hastlayer/Hast.Xilinx/Drivers/AlveoU200Driver.cs

```diff
@@ -1,15 +1,35 @@
 using Hast.Layer;
-using Hast.Synthesis.Abstractions;
-using Hast.Xilinx.Abstractions.Helpers;
+using Hast.Synthesis;
+using Hast.Synthesis.Models;
+using Hast.Synthesis.Services;
+using Hast.Xilinx.Helpers;
 using static Hast.Common.Constants.DataSize;
 using static Hast.Common.Constants.Frequency;
 
-namespace Hast.Xilinx.Abstractions.ManifestProviders;
+namespace Hast.Xilinx.Drivers;
 
-public class AlveoU200ManifestProvider : IDeviceManifestProvider
+public class AlveoU200Driver : IDeviceDriver
```

Comment:
Yes, internally it's not worth the large effort to change it.
As for library consumers, since Hastlayer uses its own service provider with a distinct lifecycle it could be confusing to expect something `Startup.ConfigureServices` style if the host application is already ASP.Net Core or OC. Devs who want to directly register their own services MS DI style can do it in the `IHastlayerConfiguration.OnServiceRegistration(configuration, services)` action.

Reply:
OK then.